### ISSUE TITLE
Enhancement: Decreasing ranges

### ISSUE THREAD

**Issue by acarabott** [armadillu/ofxRemoteUI#13]
Often it's useful to have ranges that decrease, with the highest value on the left and the lowest on the right, currently this isn't possible with the OS X client.

My use case: controlling a pixelation shader where a high value (lots of pixels) has low visual effect, but a low value (few pixels) has a high visual effect.

**Comment by armadillu** [comment]
I agree that it would be nice, but min < max assumption is all over the place... It can probably be done but not sure it's worth the amount of work and testing right now...
